### ISSUE TITLE
Accessibility Reporting

### ISSUE THREAD

**Issue by bbertucc** [joinedupack/cloud#13]
What real problem does this feature solve?

Universities create a vast number of websites, often managed by a similarly vast number of different staff from across the institution. Accessibility of those sites is often unconsidered due to budget and timeline constraints, and in many cases, a lack of awareness and understanding.

As a result, sites are published that fail to comply with accessibility guidelines. Thus, putting university reputations at risk. Some US universities have even had legal action taken against them as a result of inaccessible web properties.

Higher Ed Pros want WordPress to be more accessible. This includes keeping published content accessible and the WordPress UX accessible.

The 30 folks who are interested in “edupack accessibility” all hope our solution is accessible to WCAG AA standards and makes WordPress more accessible.
How does the feature meet our company’s mission - improving the efficiency of Higher Ed website management?

Accessibility reporting across all university websites in a single place would help to raise awareness of compliance issues more quickly ensuring corrective measures can be taken.

Technology can only go part of the way to ensuring web experiences fit for all. Human intervention is required. However, many university staff are unaware of accessibility, the guidelines and how to ensure their site remains compliant. Edupack could offer a way to educate new users during the onboarding process.
Without this feature, what alternatives do users resort to? List any competing products.

There are many players in this field already. Little Forest being one example as well as Siteimprove.

Also, we don’t have a clear picture of what accessibility features we need to build other than keeping our product accessible.
Do our economics work?

Probably not. With such tough competition, particularly from those already established in the HE space, it’s difficult to see the value in accessibility as a viable feature. Perhaps an integration with a third party vendor may be a more appropriate avenue to explore.

Given previous conversations with Gavin from Little Forest regarding preferential rates for Edupack customers, this may be a good opportunity to save Edupack customers money and further enhance our sell.
What will people pay for our feature?
What work is it going to take?
Does this feature rely on another UI (like WP or Little Forrest)?

The feature needs scoping in more detail to answer this question accurately.
Anything else to consider?

The feature needs scoping in more detail to answer this question accurately.
Why is this feature accepted or rejected?

There are too many competitors in the space. We will eventually revisit this feature idea when we are ready for integrations.
